bokeh/models/sources.py

```diff
@@ -283,6 +283,12 @@ class AjaxDataSource(RemoteSource):
     to the server. If this header is supported by the server, then only
     new data since the last request will be returned.
     """)
+    content_type = String(default='application/json', help="""
+    Set the contentType parameter in the ajax call.
+    """)
```

Comment:
Is there a finite list of acceptable content types, or is the value arbitrary? If there is a list, we can make an enumeration for it.